po/fr.po

```diff
@@ -681,60 +679,55 @@ msgid "Preferences"
 msgstr "Préférences"
 
 msgid "Installation options:"
-msgstr ""
+msgstr "Options d'installation :"
 
 msgid "Try to remove libraries before (re)installing them?"
-msgstr ""
+msgstr "Essayer de supprimer les bibliothèques avant de les (ré)installer ?"
 
 msgid "Show README of newly installed libraries (if present)?"
-msgstr ""
+msgstr "Afficher le README (si présent) des nouvelles bibliothèques ?"
 
 msgid "Should newly installed libraries be added to Pd's search path?"
-msgstr ""
+msgstr "Ajouter les nouvelles bibliothèques aux chemins de recherche de Pd ?"
```

Comment:
Suggesting : "les nouvelles bibliothèques" -> "les nouvelles bibliothèques installées"

Suggesting : "Est-ce que les nouvelles bibliothèques installées devraient être ajoutées aux chemins de recherche de Pd ?"

Reply:
idem, and I ended with: "Les bibliothèques nouvellement installées doivent-elles être ajoutées aux chemins de recherche de Pd ?"

Reply:
Fine.